### PR TITLE
Build 0.10.1

### PR DESCRIPTION
**Pull request by effigies**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by effigies** [comment]
@conda-forge-admin, please rerender

**Comment by effigies** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/nipy/nitime/archive/refs/tags/rel/{{ version }}.tar.gz
-  sha256: badf4b5b6940f339c39a2fa873b1dcb44e41f4294f72a0e1d163881a1f973a69
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (effigies) [inline recipe/meta.yaml:10]: Increasingly the convention to use PyPI sdists as the source for wheels, conda packages and Linux distro packages.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,18 +1,19 @@
 {% set name = "nitime" %}
-{% set version = "0.9" %}
+{% set version = "0.10.1" %}
+{% set sha256 = "367a15ad2b7a3134dc36ea757beff5bf92681c261cc9cb901f8ac72f35c9b7e6" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: We don't need a single use jina variable.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/nipy/nitime/archive/refs/tags/rel/{{ version }}.tar.gz
-  sha256: badf4b5b6940f339c39a2fa873b1dcb44e41f4294f72a0e1d163881a1f973a69
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: This is old style and not needed.

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml:10]: Not really. We use the best source available. Sometimes the PyPI sdist is missing files, license, etc. If that is not the case here we can use the PyPI sdist, no problem.

Comment (effigies) [reply recipe/meta.yaml:10]: I was guessing the reason the autotick bot didn't attempt to upgrade was because it was pointing to a GitHub archive instead of PyPI. Do you know the rules?

In any case, the sdist does contain the license, and the package is PEP-517 compliant so wheels are now generated from the sdist.

Comment (ocefpaf) [reply recipe/meta.yaml:10]: > I was guessing the reason the autotick bot didn't attempt to upgrade was because it was pointing to a GitHub archive instead of PyPI. Do you know the rules?

If there are no new tags on GH that may be the cause but this tag is there, see https://github.com/nipy/nitime/releases/tag/0.10.1. However, I checked the [conda-forge status page](https://conda-forge.org/status/) and it seems that the bot is not smart enough to move to a tag URL when a release URL is expected. See

Screenshot from 2023-05-24 14-24-17


TL;DR moving the the PyPI one here is fine but we, conda-forge, should make this more flexible and take both tag and releases URLs for GH udpates.

**Comment by ocefpaf** [comment]
@effigies do you want to add yourself as a maintainer here before we merge this one?

**Comment by effigies** [comment]
Sure. One moment.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender
